File: 33. Search in Rotated Sorted Array/Solution.java
public class Solution {

    public int search(int[] nums, int target) {

        for(int i = 0; i < nums.length; i++){
            if (target == nums[i]) {
                return i;
            }
        }
        
        return -1;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] nums = {2,3,4,0,1};
        int target = 13;
        System.out.println(solution.search(nums, target));
    }
    
}

File: 33. Search in Rotated Sorted Array/solution.js

nums = [4,5,6,7,0,1,2];
target = 6;

var search = function(nums, target) {
    for (let i = 0; i < nums.length; i++) {
        if (target === nums[i]) {
            return i;
        }
        
    }
    return -1;
}

console.log(search(nums,target));